admin/query/pom.xml

```diff
@@ -70,18 +70,6 @@
 
```

Comment:
Should this pom still have `<packaging>bundle</packaging>` on line 25?

Reply:
I didn't look at those modules since we've removed them from the build for now, but that will need to be removed. I'll just remove them in my next PR since there's no rush.